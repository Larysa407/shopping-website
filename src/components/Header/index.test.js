import React from "react";
import { Header } from "./index";
import "@testing-library/jest-dom/extend-expect";
import { BrowserRouter as Router } from "react-router-dom";
import renderer from "react-test-renderer";
import { Provider } from "react-redux";
import store from "../../redux/store/index"

describe("Header component", () => {
    test("snapshot test for Header component", () => {
        const component = renderer.create(
          <Provider store={store}>
            <Router>
              <Header />
            </Router>
          </Provider>
        );
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
      });
})