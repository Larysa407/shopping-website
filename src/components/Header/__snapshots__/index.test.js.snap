// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component snapshot test for Header component 1`] = `
<div
  className="header"
>
  <div
    className="links"
  >
    <a
      className="header-brand"
      href="/"
      onClick={[Function]}
    >
      Brand's Outlet
    </a>
    <div
      className="header-icons"
    >
      <a
        className="favorite-link"
        href="/favorite"
        onClick={[Function]}
      >
        <svg
          fill="currentColor"
          height={25}
          viewBox="0 0 256 256"
          width={25}
          xmlns="http://www.w3.org/2000/svg"
        >
          <rect
            fill="none"
            height="256"
            width="256"
          />
          <path
            d="M132.4,190.7l50.4,32c6.5,4.1,14.5-2,12.6-9.5l-14.6-57.4a8.7,8.7,0,0,1,2.9-8.8l45.2-37.7c5.9-4.9,2.9-14.8-4.8-15.3l-59-3.8a8.3,8.3,0,0,1-7.3-5.4l-22-55.4a8.3,8.3,0,0,0-15.6,0l-22,55.4a8.3,8.3,0,0,1-7.3,5.4L31.9,94c-7.7.5-10.7,10.4-4.8,15.3L72.3,147a8.7,8.7,0,0,1,2.9,8.8L61.7,209c-2.3,9,7.3,16.3,15,11.4l46.9-29.7A8.2,8.2,0,0,1,132.4,190.7Z"
            fill="none"
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
            strokeWidth="16"
          />
        </svg>
        <div
          className="favorite-length"
        >
          0
        </div>
      </a>
      <a
        className="cart-link"
        href="/cart"
        onClick={[Function]}
      >
        <svg
          fill="currentColor"
          height={25}
          viewBox="0 0 256 256"
          width={25}
          xmlns="http://www.w3.org/2000/svg"
        >
          <rect
            fill="none"
            height="256"
            width="256"
          />
          <path
            d="M184,184H69.8L41.9,30.6A8,8,0,0,0,34.1,24H16"
            fill="none"
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
            strokeWidth="16"
          />
          <circle
            cx="80"
            cy="204"
            fill="none"
            r="20"
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
            strokeWidth="16"
          />
          <circle
            cx="184"
            cy="204"
            fill="none"
            r="20"
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
            strokeWidth="16"
          />
          <path
            d="M62.5,144H188.1a15.9,15.9,0,0,0,15.7-13.1L216,64H48"
            fill="none"
            stroke="currentColor"
            strokeLinecap="round"
            strokeLinejoin="round"
            strokeWidth="16"
          />
        </svg>
        <div
          className="cart-length"
        >
          0
        </div>
      </a>
    </div>
  </div>
</div>
`;
